libs/SalesforceSDK/AndroidManifest.xml

```diff
@@ -25,8 +25,39 @@
         </service>
 
         <!-- Login activity -->
+        <!--
+            Launch mode of "singleInstance" ensures that the activity isn't restarted
+            by a callback from Chrome custom tab when auth flow is complete. This is
+            required for the Chrome custom tab auth flow to work correctly.
+        -->
         <activity android:name="com.salesforce.androidsdk.ui.LoginActivity"
-            android:theme="@style/SalesforceSDK.ActionBarTheme" />
+            android:theme="@style/SalesforceSDK.ActionBarTheme"
+            android:launchMode="singleInstance">
+
+            <!--
+                To enable browser bath authentication, uncomment the lines below and replace
+                'scheme', 'host' and 'path' with their corresponding values from your connected app.
+
+                For example, if the callback URL of your connected app is
+                "testsfdc:///mobilesdk/detect/oauth/done",
+                'scheme' would be "testsfdc", 'host' would be "*" since it doesn't exist, and
+                'path' would be "/mobilesdk/detect/oauth/done".
+
+                If the callback URL is "sfdc://login.salesforce.com/oauth/done",
+                'scheme' would be "sfdc", 'host' would be "login.salesforce.com",
+                and 'path' would be "/oauth/done".
+            -->
+            <!--
+            <intent-filter>
```

Comment:
An `Activity` needs to be registered to receive an `Intent` when `callbackUri` is triggered. It needs to be broken down by `host`, `scheme` and `path`.